services/cdnjs/cdnjs.tester.js

```diff
@@ -31,7 +31,7 @@ t.create('cdnjs (unexpected response)')
     .get('/libraries/jquery?fields=version')
     .reply(invalidJSON)
   )
-  .expectJSON({name: 'cdnjs', value: 'invalid'});
+  .expectJSON({ name: 'cdnjs', value: 'unparseable json response' });
```

Comment:
At this point, if we could get some coverage of this path in the BaseJsonService tests, I'd be comfortable getting rid of all these `invalidJSON` tests on new-style services. Thoughts?

Reply:
Good point. I reckon for badges/services that just use standardized error handling provided by the base classes we can dispense with testing the standard error cases in every service tester. Where we've done something custom we should still test that.

If we are going to do that, can we make sure those standard cases are covered in `base.spec.js`.

Reply:
Have updated this, removing those test cases and adding a new test for BaseJsonService.

As `base.js` is getting long, I'm thinking we probably should break out BaseJsonService to its own module, so I put the tests in a new file.